### PR TITLE
Get postgres data from react

### PR DESCRIPTION
**Pull request by jasontyu**
* installs `dotenv`
* updates React `index.js` with proof of concept API call and render of `postgres` data  

Notes: make sure to `npm install` and update `env.example` to `.env` after pulling

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+//           href="https://reactjs.org"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
```
Comment (jasontyu) [inline client/src/App.js:29]: Delete comments

File: client/src/App.js [hunk 2]
```diff
@@ -1,26 +1,73 @@
...
+
+class getPostgres extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { persons: ['placeholder name']}
```
Comment (jasontyu) [inline client/src/App.js:35]: [nit] Missing space before `}`

File: db/postgres.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const { Client } = require('pg');
+
+// console.log(process.env.DB_LOCAL_DATABASE);
+// console.log(process.env);
```
Comment (jasontyu) [inline db/postgres.js:4]: [nit] Remove comments

File: db/postgres.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const client = new Client({
+    user: process.env.PG_LOCAL_USER,
+    host: process.env.PG_LOCAL_HOST,
+    database: process.env.PG_LOCAL_DATABASE,
+    port: process.env.PG_LOCAL_POST
+});
+
+client.connect();
+
+// console.log(client);
```
Comment (jasontyu) [inline db/postgres.js:15]: [nit] remove comment

File: client/src/App.js [hunk 5]
```diff
@@ -1,26 +1,73 @@
...
+                console.log('componentDidMount: axios.get(/test).then');
+                console.log(res.data);
+                const persons = res.data;
+                this.setState({
+                    persons: persons.map(person => person.firstname)
+                });
+        })
+    }
+
+    // componentDidMount() {
```
Comment (LeekSwan) [inline client/src/App.js:52]: [nit] Delete comments

File: client/src/index.js [hunk 6]
```diff
@@ -5,14 +5,31 @@ import './index.css';
...
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById('root')
+// );
+
+// // If you want your app to work offline and load faster, you can change
+// // unregister() to register() below. Note this comes with some pitfalls.
+// // Learn more about service workers: https://bit.ly/CRA-PWA
+// serviceWorker.unregister();
+
```
Comment (LeekSwan) [inline client/src/index.js:19]: [nit] delete comments

File: routes/index.js [hunk 7]
```diff
@@ -1,18 +1,23 @@
...
 // })
 
 // define the about route
-router.get('/:userId', function (req, res) {
-
-
-
+// router.get('/:userId', function (req, res) {
+//   res.send('user Id')
+// })
```
Comment (LeekSwan) [inline routes/index.js:13]: [nit] delete comments

**Pull request merged**